job_definitions/notify_suppliers_of_framework_clarification_questions_answers.yml

```diff
@@ -0,0 +1,52 @@
+{% set environments = ['preview', 'production'] %}
+{% set notify_template_id = 'ed9944eb-112e-4f04-a4bb-19f397550ffe' %}
+{% set framework_slug = 'g-cloud-12' %}
```

Comment:
Could these be job parameters instead of being hardcoded into the template?

Reply:
Yep, defaults could work. I'll have a think...

Reply:
Have pushed a commit 👍